util/avatars.ts

```diff
@@ -0,0 +1,7 @@
+export const githubProfileAvatarUrl = (username: string): string =>
+  `https://avatars.githubusercontent.com/${username}`;
+
+export const getAcronymFromString = (str = '') =>
+  str.length > 0
```

Comment:
Why are you checking `str.length`? The string should always have something.

Reply:
> Why are you checking str.length? 

`str.length` it's just to output something if string it's empty

Reply:
But we shouldn't provide empty strings here, that's the point.

Reply:
I don't really get your idea.
so you mean do something like that:
```js
export const getAcronymFromString = (str: string) =>
  str.length > 0 || str
    ? [...(str.trim().match(/\b(\w)/g) || '')].join('').toUpperCase()
    : 'NA';
```